src/engine/SCons/Action.py

```diff
@@ -116,8 +115,7 @@
 import SCons.Subst
 
 # we use these a lot, so try to optimize them
-is_String = SCons.Util.is_String
-is_List = SCons.Util.is_List
+from SCons.Util import is_String, is_List
```

Comment:
Does this yield the same symbol lookup overhead?

Reply:
I don't know. I know imported symbol like `is_String` is microscopically faster than module-qualified like `SCons.Util.is_String` (to the point where "nobody should care about the difference"), but I don't know how that compares to a local variable bound to the qualified name. I'd guess the latter is a tiny bit faster, but I read the "optimize" here as reducing typing, not affecting performance.